Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Modules\CodeTrek\Services;
+
+use Modules\CodeTrek\Entities\CodeTrekApplicant;
+
+class CodeTrekService
+{
+    public function getCodeTrekApplicants()
+    {
+        $applicants = CodeTrekApplicant::all();
+
+        return ['applicants'=> $applicants];
+    }
+    public function store($data)
+    {
+        $applicant = new CodeTrekApplicant();
+
+        $applicant->first_name = $data['first_name'];
+        $applicant->last_name = $data['last_name'];
+        $applicant->email = $data['email_id'];
+        $applicant->phone = $data['phone'];
```

Comment:
Use null coalescing in fields that are nullable
```suggestion
        $applicant->phone = $data['phone'] ?? null;
```